### ISSUE TITLE
Edit custom notification for foreground service

### ISSUE THREAD

**Issue by agnoam** [ekasetiawans/flutter_background_service#450]
Hi, 
I'm using this package for implementing a foreground service, it works well.
But I need to show up a notification with an action in it. I implemented it by myself with [flutter_local_notifications](https://pub.dev/packages/flutter_local_notifications). 
But it seems that I can't just hide this package notification without using it as a background service, which will make the OS able to kill this service periodically.

So to summarize, how can I show my custom notification (with actions) instead of the default one provided by the package?
